### ISSUE TITLE
BatchNormalization

### ISSUE THREAD

**Issue by Stick-To** [tensorflow/tensorflow#34009]
If I want to freeze gamma, beta, moving_average_mean, moving_average_var in BatchNormalization
could tf.keras.layers.BatchNormalization(trainable=False)  be OK?
or only could freeze gamma, beta ?

**Comment by ymodak** [comment]
```trainable``` attribute in BatchNormalization disables the updates of the batch statistics
https://www.tensorflow.org/api_docs/python/tf/keras/layers/BatchNormalization#__init__
